### ISSUE TITLE
Use standard constructors for std::vector

### ISSUE THREAD

**Issue by exfret** [maxsu/balancers2#8]
There is a standard std::vector constructor to create a vector with N zeroes (or even N of a given element). I think we should actually be using this instead (and maybe look into a special vector-like class that someone has already implemented that would be good since it might also support multi-threading and such).

**Comment by maxsu** [comment]
This is low hanging fruit. I'm promoting it to the current sprint.

Dependencies:

- [ ] Merge PR #16, so we can modify `constRow` directly.

Tasks: 

- [ ] Modify `constRow`
- [x] Scan for other uses of pushBack to create constant vectors
  - [ ] exists_balancer.cpp:L116-123 
    May need a `constRowMatrix` function, eg,
    ```cpp
    Matrix balancer = constRowMatrix(output_size, constRow(input_size, 1.0 / output_size));
    ```
    Although the native syntax is tempting tbh:
    ```cpp
    Matrix balancer = Matrix(output_size, Row(input_size, 1.0 / output_size));
    ```
  - [ ] network_tools.cpp:L199. May be obviated by PR #16.

**Comment by maxsu** [comment]
Fixed in #22
